packages/react-charting/src/components/CommonComponents/error-utils.ts

```diff
@@ -0,0 +1,11 @@
+import { IChartProps, IHorizontalBarChartWithAxisDataPoint } from '../../index';
+
+export const isHorizontalBarChartEmpty = (data: IChartProps[] | undefined): boolean => {
+  return !(data && data.length > 0);
+};
+
+export const isHorizontalBarChartWithAxisEmpty = (
```

Comment:
as I mentioned earlier keep this logic in the corresponding chart only

Reply:
Since we are wrapping the base class, how can we backward propagate a handler?

Reply:
use a callback function as prop.
https://medium.com/@amitsharma_24072/passing-functions-as-props-in-react-harnessing-the-power-of-callbacks-dafe08ba0edb

Reply:
this comment is still not resolved.